src/testsuite/vec_pwr10_dummy.c

```diff
@@ -0,0 +1,124 @@
+/*
+ Copyright (c) [2020] IBM Corporation.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+
+ vec_pwr10_dummy.c
+
+ Contributors:
+      IBM Corporation, Steven Munroe
+      Created on: Nov. 24, 2020
+ */
+
+#if __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
+//#pragma GCC target ("cpu=power10")
+#include <stdint.h>
+#include <stdio.h>
+#include <fenv.h>
+#include <float.h>
+#ifndef PVECLIB_DISABLE_F128MATH
+/* Disable for __clang__ because of bug involving <math.h>
+   incombination with -mcpu=power9 -mfloat128 */
+#include <math.h>
+#endif
+
+#include <pveclib/vec_int128_ppc.h>
+#include <pveclib/vec_int512_ppc.h>
+#include <pveclib/vec_f128_ppc.h>
+#include <pveclib/vec_f32_ppc.h>
+#include <pveclib/vec_bcd_ppc.h>
+
+vui128_t
+__test_muloud_PWR9 (vui64_t a, vui64_t b)
```

Comment:
Is it intentional these functions all have suffix `_PWR9`?

Reply:
Yaaa  no. This will not matter until I commit the operation changes for P10. At which point I would expects to get duplicate symbols on the link.

So will changes these all to _PWR10 suffix.